### DISCUSSION TITLE
Connection Refused

### DISCUSSION THREAD

**Discussion by AntonyMoraes** [opencardev/crankshaft#611] [General]
Hey there, may someone help me?

I'm getting Connection Refused when I trie to connect AA Wireless

![WhatsApp Image 2023-01-03 at 17 04 19](https://user-images.githubusercontent.com/121897534/210432788-33c032a2-9a71-4af5-9993-9fb7bf8d4f08.jpeg)

**Comment by Geekyadz** [comment]
Have you enabled developer mode in Android Auto?

![Screenshot_20230106-133945](https://user-images.githubusercontent.com/30429165/211023604-c9788e82-c2bd-488a-830c-6b1a6a632744.png)
